Makefile

```diff
@@ -1,10 +1,15 @@
 RUNC_TEST_IMAGE=runc_test
 PROJECT=github.com/opencontainers/runc
 TEST_DOCKERFILE=script/test_Dockerfile
+BUILD_TAGS=seccomp
```

Comment:
maybe `BUILDTAGS` to be consistent with `TESTFLAGS`, also underscore is superhard to type :)